src/rules/no_mixed_spaces_and_tabs.rs

```diff
@@ -89,6 +89,34 @@ impl LintRule for NoMixedSpacesAndTabs {
       }
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows mixed spaces and tabs for indentation.
+
+Most code rules need to use of tabs or spaces for indentation. Therefore, if a line of code is indented with both tabs and spaces, it will usually result in an error.
+
+### Invalid:
+
+```typescript
+function add(x, y) {
+      return x + y;
+}
```

Comment:
It would be much nicer to add a comment that describes the line of `return` is indented with both a tab and spaces, because it's hard to recognize by just looking at the rendered snippet.

And it looks like the line of `return` is actually indented with only spaces, doesn't have tab character. Your intention may be like the following?

```ts
function add(x: number, y: number) {
	  return x + y; // indented with a tab + two spaces
}
```

(type annotations for parameters would be fine as well since it's TypeScript)